tests/Oryx.Tests.Common/ImageTestHelper.cs

```diff
@@ -319,9 +333,10 @@ public string GetGitHubActionsAsBaseWithEnvBuildImage(string debianFlavor = null
             }
         }
 
-        public string GetVsoBuildImage(string debianFlavor=null)
+        public string GetVsoBuildImage(string debianFlavor = null)
         {
-            return $"{_repoPrefix}/{_buildRepository}:{_vsoUbuntu}{_tagSuffix}";
+            string _vsoImage = !string.IsNullOrEmpty(debianFlavor) && debianFlavor == "bullseye" ? _vsoBullseye : _vsoUbuntu;
```

Comment:
Can we make this consistent with the other test methods? They have the image test helper constants (e.g. ImageTestHelperConstants.VsoBullseye) passed in as the `debianFlavor` parameter, instead of the actual debian flavor. This naming can probably be updated as its confusing. 

They then check to see if what is passed in matches the constant e.g.
`string.Equals(debianFlavor.ToLower(), _azureFunctionsJamStackBullseye)` which in this case would be `_vsoBullseye` or `_vsoUbuntu`

All of this allows us to only have to update the `ImageTestHelperConstants` value when an image tag changes, instead of find every place that `GetVsoBuildImage` or `GetBuildImage` is used and seeing if it also needs to be changed

Reply:
Agreed, I've updated this method.